presto-thrift-interface/src/main/java/com/facebook/presto/thrift/interfaces/readers/ArrayColumnReader.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.thrift.interfaces.readers;
+
+import com.facebook.presto.spi.block.ArrayBlock;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.thrift.interfaces.client.ThriftColumnData;
+import com.facebook.presto.thrift.interfaces.readers.IntColumnReader.NullsAndOffsets;
+
+import java.util.List;
+
+final class ArrayColumnReader
+{
+    private ArrayColumnReader()
+    {
+    }
+
+    public static Block readBlock(List<ThriftColumnData> columnsData, String columnName, Type columnType, int totalRecords)
+    {
+        NullsAndOffsets nullsAndOffsets = IntColumnReader.readNullsAndOffsets(columnsData, columnName, totalRecords);
```

Comment:
The method is in `IntColumnReader` because it reads data as ints and shares some logic with `readBlock` in `IntColumnReader`.